pulsar-client-cpp/python/pulsar/__init__.py

```diff
@@ -472,7 +472,8 @@ def subscribe(self, topic, subscription_name,
                   broker_consumer_stats_cache_time_ms=30000,
                   is_read_compacted=False,
                   properties=None,
-                  pattern_auto_discovery_period=60
+                  pattern_auto_discovery_period=60,
+                  initial_position=None
```

Comment:
The default here is `Latest` anywyay, so we could set it to that instead of None